x/resource/genesis.go

```diff
@@ -33,15 +37,18 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState *types.GenesisState)
 }
 
 // ExportGenesis returns the cheqd module's exported genesis.
-func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *types.GenesisState {
+func ExportGenesis(ctx context.Context, k keeper.Keeper) *types.GenesisState {
 	// Get all resource
-	resourceList, err := k.GetAllResources(&ctx)
+	resourceList, err := k.GetAllResources(ctx)
 	if err != nil {
 		panic(fmt.Sprintf("Cannot get all resource: %s", err.Error()))
 	}
 
 	// get fee params
-	feeParams := k.GetParams(ctx)
+	feeParams, err := k.GetParams(ctx)
+	if err != nil {
+		panic(fmt.Sprintln("Cannot get fee params: %s", err.Error()))
```

Comment:
Again, as denoted: Let's either replace with a valid syntactical format or skip the lint rule.